compiler/tflchef/core/src/DataChef.def

```diff
@@ -25,6 +25,10 @@ DATA_CHEF(INT16, gaussian, GaussianInt16DataChefFactory)
 DATA_CHEF(INT8, gaussian, GaussianInt8DataChefFactory)
 DATA_CHEF(UINT8, gaussian, GaussianUint8DataChefFactory)
 
+// INT4 support only for constant, explicit as int8_t
+DATA_CHEF(INT4, constant, ConstantDataChefFactory<int8_t>)
+DATA_CHEF(INT4, explicit, ExplicitDataChefFactory<int8_t>)
```

Comment:
This would create int8 tensor even though the dtype is int4. AFAIK, int4 values have to be packed in tflite (or circle)?

Reply:
yes, this will create int8_t in memory, and then packed when creating tflite flatbuffer file.